.gitignore

```diff
@@ -1,2 +1,4 @@
 .idea/
 **/.DS_Store
+/out/
+/java/mykola-melnyk/src/Draft/
```

Comment:
that's a global .gitignore file, please create a local one inside of your project

Reply:
I have restored the specified .gitignore file.